### DISCUSSION TITLE
Trend in version upgrade

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#1681] [General]
Bonjour,

I am curious as to what is rate of newer version of Jamulus (client) adoption ?  There should be statistics available to maintainers / developers from the download pages.

As many users (particularly Microsoft users) know, it is mandatory to be cautious about versions of software terminating with ".0".  This led me to collect some statistics on 3.7.0 adoption, although only on the server side via explorer.jamulus.io .

For anyone interested , I offer this spreadsheet in Excel [ Microsoft Excel 97/2000/XP (.xls) ] .
[Jamulus 3-7-x upgrade trends.xls](https://github.com/jamulussoftware/jamulus/files/6476022/Jamulus.3-7-x.upgrade.trends.xls)

**Comment by hoffie** [comment]
> There should be statistics available to maintainers / developers from the download pages.

I think we only have those on the non-primary download location at SourceForge. @gilgongo has been watching this, I think. Also, those download numbers may underrepresent Linux users which are used to getting their software from their distributions (which often mirror the source code package or provide their own binaries).

I also think the wish for such numbers has been floating around for some time, but we should be really careful if and how such analyses can be done in a privacy-preserving way.

> As many users (particularly Microsoft users) know, it is mandatory to be cautious about versions of software terminating with ".0".

Yes, I know this from Enterprise software as well, but this should really be about .0 major versions (e.g. some potential 4.0), but not about minor versions (3.7.0). To make release planning more consistent and to be able to do bugfix-only releases while clearly marking them, we're focusing on bumping the minor version for new features from now on. The upcoming release will be 3.8.0 (= patch level 0) and may stay at that stage if no bugfixes are necessary.

> For anyone interested , I offer this spreadsheet in Excel [ Microsoft Excel 97/2000/XP (.xls) ] .

Thanks! I'm wondering whether @softins may have similar statistics from the Explorer backend?
@pljones as the maintainer of several directory servers did also have a frequent look at those numbers, I think.

**Reply by gilgongo** [reply]
There are general download stats [on SourceForge](https://sourceforge.net/projects/llcon/files/stats/timeline) (note that we made GitHub the primary download in March), but for some reason the version breakdown seems not to be accurate.

This seems to be [a better source of data](https://somsubhra.github.io/github-release-stats/?username=jamulussoftware&repository=jamulus&page=1&per_page=5):
